test/import/FSHImporter.test.ts

```diff
@@ -83,21 +83,6 @@ describe('FSHImporter', () => {
     assertAssignmentRule(profile.rules[2], 'bonusCode', expectedBonusCode);
   });
 
-  it('should parse a rule with an identifying integer', () => {
```

Comment:
Did we lose this feature?  Or just the test?

Reply:
This feature had to go because it worked like this:
```
*123 name.family
```
So the `123` had to immediately follow the `*`, or else it may as well be a path. I guess now that I think about it, I probably could allow the optional numbers after the `*`, and then require a space after the end of the numbers. But we are already making breaking changes, to me this seems like a good time to get rid of this feature. At least according to regression, it is completely unused. I know it does no harm to support this...but we already enforce good style in other places, so to me it seems like we should enforce good style here by not adding a feature that to me seems like a bad style choice.

Reply:
Seeing as you ran this by Mark K and he also indicated it could be removed, I'm fine with it too.